lib/containers/common.pm

```diff
@@ -240,6 +243,7 @@ sub test_container_image {
             upload_logs("$logfile");
             die "Heartbeat test failed for $image";
         }
+        assert_script_run "rm $logfile";
```

Comment:
let's keep context here . we are speaking about openQA test which run on clone of qcow image which would be deleted after run , why we need to bother with such things at all ?

Reply:
For bigger tests (e.g. glibc 32bit, salt), cleanup is absolutely necessary to conserve resources. For others, it's necessary to avoid polluting the environment, so that needles and other stuff still works as expected.